src/main/java/net/spy/memcached/ClientMode.java

```diff
@@ -19,5 +19,9 @@ public enum ClientMode {
    * The client is initialized with a configuration endpoint. The client will periodically learn about the cache nodes in the
    * cluster.
    */
-  Dynamic
+  Dynamic,
+  /**
+   * Default client mode, we will perform mode discovery based on the endpoint dns name.
+   */
+  Unset
```

Comment:
Move this up to be the first entry as this is the default. i.e. before `Static`.